packages/client/src/Config.ts

```diff
@@ -50,64 +50,241 @@ export interface EthereumNetworkConfig {
 export interface StreamrClientConfig {
     /** Custom human-readable debug id for client. Used in logging. */
     id?: string
+
+    /**
+     * Override the default logging level.
+     */
     logLevel?: LogLevel
+
     /**
-    * Authentication: identity used by this StreamrClient instance.
-    * Can contain member privateKey or (window.)ethereum
+    * The Ethereum identity to be used by the client. Either a private key
+    * or a window.ethereum object.
     */
     auth?: PrivateKeyAuthConfig | ProviderAuthConfig
 
-    /** Attempt to order messages */
+    /**
+     * Set to true to enable message ordering.
+     *
+     * Due to the distributed nature of the network, messages may occasionally
+     * arrive to the client out-of-order. By enabling message ordering, the
+     * client will reorder received messages to the intended order for you.
+     *
+     * */
```

Comment:
The first line is kind of trivial (guessable from the field name). We could consider if it is more straightforward to write this without introducing thee extra term. 

I.e. skip the middle step in this chain: "I want to client to reorder messages" -> "I need to enable message ordering" -> "I set this to `true`".

```suggestion
    /**
  
     * Due to the distributed nature of the network, messages may occasionally
     * arrive to the client out-of-order. Set this option to `true` if you want that the 
     * client reorders the received messages to the intended order.
     *
     * */
```